pkgs/applications/misc/openrgb/default.nix

```diff
@@ -17,6 +17,9 @@ stdenv.mkDerivation rec {
   installPhase = ''
     mkdir -p $out/bin
     cp openrgb $out/bin
+
+    mkdir -p $out/etc/udev/rules.d
+    cp 60-openrgb.rules $out/etc/udev/rules.d
```

Comment:
we should probably make this into a module, and allow for this to be setup correctly for NixOS

but can be done in a later PR